test/check-realms

```diff
@@ -141,7 +141,7 @@ for x in 1 2 3 4 5 6 7 8 9 10; do
     if nslookup -type=SRV _ldap._tcp.cockpit.lan; then
         break
     else
-        sleep 1
+        sleep 5
```

Comment:
If we use sleep x here we have a nice backoff and about the same total waiting time in case of a failure while keeping response time low for fast systems.